custom/icds_reports/views.py

```diff
@@ -1566,3 +1605,23 @@ class ICDSBugReportView(BugReportView):
     @property
     def recipients(self):
         return [ICDS_SUPPORT_EMAIL]
+
+
+@method_decorator([login_and_domain_required], name='dispatch')
+class DownloadPDFReport(View):
+    def get(self, request, *args, **kwargs):
+        uuid = self.request.GET.get('uuid', None)
+        format = self.request.GET.get('format', None)
+        pdf_dir = os.path.join(settings.BASE_DIR, 'custom/icds_reports/static/media/')
```

Comment:
as mentioned already above, this won't work. we'll need to use a storage that isn't tied to the machine/os.